crosstool/BUILD.toolchains

```diff
@@ -0,0 +1,100 @@
+package(default_visibility = ["//visibility:public"])
+
+# Target constraints for each arch.
+# TODO(apple-rules): Rename osx constraint to macOS.
```

Comment:
Is there migration needed for this, or what is preventing us from doing this now?

Reply:
don't think so anymore since @platforms was forced on folks and they're an alias here, i just copied this one without modification